docs/modules/sql/pages/get-started-sql.adoc

```diff
@@ -65,188 +70,206 @@ You should see the following:
+. Use another filter to display only city names beginning with the letter L where the length is > 6.
 +
 [source,sql]
 ----
-SELECT __key AS countries
+SELECT cities AS City
 FROM cities
-WHERE LENGTH(this) >= 6 AND __key LIKE 'C%';
+WHERE cities LIKE 'L%' AND LENGTH(cities) >6;
 ----
 +
 ```
 +--------------------+
-|countries           |
+|City                |
 +--------------------+
-|Czech Republic      |
-|Croatia             |
+|Los Angeles         |
 +--------------------+
 ```
 
 . Configure the map connector to create a new map table called `population2020`.
 +
 [source,sql]
 ----
-CREATE MAPPING population2020
-TYPE IMap OPTIONS ('keyFormat'='varchar', 'valueFormat'='int');
+CREATE MAPPING population2020 (
+__key INT,
+cities VARCHAR,
+population INT)
+TYPE IMap OPTIONS ('keyFormat'='int', 'valueFormat'='json-flat');
 ----
 
 . Add the 2020 populations of the following cities.
 +
 [source,sql]
 ----
 INSERT INTO population2020 VALUES
-('Canberra', 354644),
-('Zagreb', 804200),
-('Prague', 1227332),
-('London', 8174100),
-('Ankara', 4890893),
-('Washington, DC', 601723);
+(1, 'London', 9304016),
+(2, 'Manchester', 2730076),
+(3, 'New York', 8622357),
+(4, 'Los Angeles', 4085014),
+(5, 'Sao Paulo', 12396372),
+(6, 'Rio de Janeiro', 6775561),
+(7, 'Istanbul', 14804116),
+(8, 'Ankara', 3517182);
 ----
 
 . Use the `JOIN` clause to merge results from the `cities` and `population2020` tables so you can see which countries had the most populated captial cities in 2020.
 +
 [source,sql]
 ---- 
-SELECT cities.__key AS country, cities.this AS city, population2020.this AS population
+SELECT cities.countries AS country, cities.cities AS city, population2020.population AS population
 FROM cities
 JOIN population2020
-ON cities.this = population2020.__key;
+ON cities.cities = population2020.cities;
 ----
 +
 ```
 +--------------------+--------------------+------------+
 |country             |city                |  population|
 +--------------------+--------------------+------------+
-|England             |London              |     8174100|
-|United States       |Washington, DC      |      601723|
-|Czech Republic      |Prague              |     1227332|
-|Turkey              |Ankara              |     4890893|
-|Croatia             |Zagreb              |      804200|
-|Australia           |Canberra            |      354644|
-+--------------------+--------------------+------------+
-```
+|United Kingdom      |Manchester          |     2730076|
+|Turkey              |Ankara              |     3517182|
+|United Kingdom      |London              |     9304016|
+|Brazil              |Sao Paulo           |    12396372|
+|Brazil              |Rio de Janeiro      |     6775561|
+|Turkey              |Istanbul            |    14804116|
+|United States       |Los Angeles         |     4085014|
+|United States       |New York            |     8622357|
++--------------------+--------------------+------------+```
```

Comment:
The backticks here need to be on a new line. This format is breaking the layout.

Reply:
FIXED!